packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -555,6 +554,21 @@ class VMService implements vm_service.VmService {
     );
   }
 
+  @override
+  Future<vm_service.Isolate> getIsolate(String isolateId) {
+    return _delegateService.getIsolate(isolateId);
+  }
+
+  @override
+  Future<vm_service.Success> resume(String isolateId, {String step, int frameIndex}) {
+    return _delegateService.resume(isolateId, step: step, frameIndex: frameIndex);
+  }
+
+  @override
+  Future<vm_service.Success> kill(String isolateId) {
```

Comment:
This is the fix

Reply:
The last incremental migration is to remove the custom isolate and vm type. That will remove the need for the shimming, which will remove the need for our own vm service type, which will remove the need for the noSuchMethod